src/app.router.js

```diff
@@ -48,7 +48,20 @@ class AppRouter extends Component {
         return { baseUrl, apiVersion }
     }
 
+    getDefaultRedirect() {
+        const enforceVerifiedEmail =
+            this.props.d2.system.settings.settings.enforceVerifiedEmail
```

Comment:
I'd vote to add optional chaining here (applies to anything you've added) so that the app doesn't crash because of this info not being available 

There might be other problems that cause a crash if this info is undefined, so it probably doesn't matter much, but I think being cautious here is good given that this NoticeBox/redirect is for informational, rather that functional purposes.